.github/workflows/self-scheduled.yml

```diff
@@ -31,13 +31,12 @@ jobs:
     - name: Install dependencies
       run: |
         source .env/bin/activate
-        pip install .[sklearn,tf,torch,testing]
+        pip install .[sklearn,torch,testing]
```

Comment:
you might want to rename the job on line 12

Reply:
Also would be good to pipe stdout to a file or suppress logger.info in the code to make the tracebacks more readbale.

Reply:
Let's not rename as the goal is to re-enable the TF tests soon